pyecharts/datasets/map_filename.json

```diff
@@ -1,5 +1,8 @@
 {
   "bulma": ["bulma.min", "css"],
+  "jquery": ["jquery.min", "js"],

```

Comment:
hoho, 95% of the web uses jquery. now is 96%

Reply:
Jquery still in style